api/cpp/platform.rs

```diff
@@ -132,6 +135,22 @@ impl Platform for CppPlatform {
             invoke_from_event_loop: self.invoke_from_event_loop,
         }))
     }
+
+    fn set_clipboard_text(&self, _text: &str, _clipboard: Clipboard) {
+        let shared_text = SharedString::from(_text);
+        unsafe { (self.set_clipboard_text)(self.user_data, &shared_text, _clipboard as u8) }
+    }
+
+    fn clipboard_text(&self, _clipboard: Clipboard) -> Option<String> {
+        let mut out_text = SharedString::new();
+        unsafe { (self.clipboard_text)(self.user_data, &mut out_text, _clipboard as u8) };
+
+        if out_text.is_empty() {
```

Comment:
the callback can return a bool so we can distinguish between empty and not supported